src/status_im/contexts/communities/overview/view.cljs

```diff
@@ -191,7 +191,7 @@
            :accessibility-label :show-request-to-join-screen-button
            :customization-color color
            :icon-left :i/communities}
-          (i18n/label :t/request-to-join-community)]))
+          (i18n/label :t/request-to-join)]))
```

Comment:
The line below is suspicious too. Anything based on the `token-permissions` collection may be buggy.

https://github.com/status-im/status-mobile/blob/ed3cb4257234f0917531d64c6543587ef42ed5fb/src/status_im/contexts/communities/overview/view.cljs#L196

This other place is also suspicious:

https://github.com/status-im/status-mobile/blob/ed3cb4257234f0917531d64c6543587ef42ed5fb/src/status_im/contexts/communities/events.cljs#L75